install/scripts/knative-openshift-policies.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash 
+
+set -e 
+
+oc adm policy add-scc-to-user anyuid -z build-controller -n knative-build
+oc adm policy add-scc-to-user anyuid -z controller -n knative-serving
+oc adm policy add-scc-to-user anyuid -z autoscaler -n knative-serving
```

Comment:
@kameshsampath besides the `autoscaler` in `knative-serving`, should this be done for the `activator` as well ?

Reply:
@matzew  do you see any issues because of it ?

Reply:
I dont see an SA with `autoscaler` .. where do you see it?